lib/exchange.js

```diff
@@ -34,9 +34,16 @@ export async function generateResponse({
   return {verifiablePresentationRequest};
 }
 
-// eslint-disable-next-line no-unused-vars
+/**
+ * If an exchangeId is defined in the config return true.
+ *
+ * @param {object} options - Options to use.
+ * @param {string} options.exchangeId - The id of an exchange.
+ *
+ * @returns {boolean} Was the exchangeId in exchanges?
+ */
 export async function exists({exchangeId}) {
-  return true;
+  return (exchangeId in (bedrock.config['vc-exchanger'].exchanges || {}));
```

Comment:
Though I'm a fan of terse one liners, they can be difficult to read for new engineers.

I'd prefer the following syntax.

```javascript
const {exchanges = {}} = bedrock.config['vc-exchanger'];
return exchangeId in exchanges;
```

Normally, I hesitate to use the `in` keyword because it includes properties in the prototype chain. In this case, it should be fine.